CONTRIBUTING.md

```diff
@@ -117,6 +117,80 @@ You can learn more about [pull requests on GitHub](https://help.github.com/en/gi
 
 You can also contribute to the project without specifically writing content for the site. Many discussions are happening all around the project. Protocol Labs, and by extension, IPFS, is a collaborative company that relies heavily on the community. We need your opinions and input! If you have an idea about a topic or want to weigh in on a discussion, feel free to have your say either on [GitHub](https://github.com/ipfs/ipfs-docs/issues) or on the [IPFS forums](https://discuss.ipfs.tech/).
 
+## Create a new page
+
+Sometimes you will need to make a whole new page for content to live. The process to do this is outlined below.
+
+These steps were performed in Visual Studio Code (VSCode).
+
+The first step to creating a new page is to identify the type of content it will be to determine where it will live in the structure of the docs. It may need to be a whole new section or it could be added to an existing section as a sub-page. An important concept to keep in mind when deciding where to put the new content is the flow of the already existing content. You wouldn't want to put a page or section designed for experienced users between the already existing **Install** and **Basics** sections.
+
+Once you have determined where the content will live, you must add the appropriate lines to the `config.js` file.
+
+### Create a branch
```

Comment:
This is more of a question / discussion: should we be directing the user to just branch off the main repo and work off of that? Or should we request that they create a fork so that they can make edits there, create a PR, and then we review, merge, etc.? Just wondering if there is preference or opinions on over the other. I like forking because it means way less branches on the main repo, so my fork can be messy and it doesn't affect the overall repo quality. Would like to hear other folks take on this. @TMoMoreau I see that we already have https://docs.ipfs.tech/community/contribute/contribution-tutorial/#creating-a-fix and https://docs.ipfs.tech/community/contribute/contribution-tutorial/#create-a-pull-request. Might make sense to just tell the user to follow the create fix process, with the added note that their edits must include the necessary edits to `config.js` file (depending on whether they are adding a section or page). Then, have them follow the PR process.

Reply:
Seems like this might cause confusion, especially if a reader reads both this and the existing guidelines in the published docs.

Reply:
Unless I'm totally misunderstanding what I'm reading here (totally could be).

Reply:
Steps could be:

```
1. Complete steps 1-3 of https://docs.ipfs.tech/community/contribute/contribution-tutorial/#creating-a-fix if not already completed. Could either link the user or copy out that MD to this page.
2. Make local change. Could be broken into substeps or their individual steps: 
     1. Create page, section, grammar fix, whatever. 
     2. Make appropriate edits to config.js depending on whether it's a new section or page. 
         Could probably condense https://github.com/ipfs/ipfs-docs/pull/1294/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R146-R173 into one section. 
3. https://docs.ipfs.tech/community/contribute/contribution-tutorial/#create-a-pull-request
4. https://docs.ipfs.tech/community/contribute/contribution-tutorial/#waiting-for-a-review
5. etc.
```

Reply:
It'd be nice if contributors just made forks. Like you said @ElPaisano, individual forks can be as messy as they want, until it comes time to make a PR.

Reply:
Reader is now instructed to first fork the repo, rather than cloning the main repo. They are also directed to our existing docs on how to create a fix/PR.